umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/instrument.clj

```diff
@@ -12,6 +13,10 @@
   (str "/gmi:MI_Metadata/gmi:acquisitionInformation/gmi:MI_AcquisitionInformation/gmi:instrument"
     "/eos:EOS_Instrument"))
 
+(def instrument-alternative-xpath
+  (str "/gmi:MI_Metadata/gmi:acquisitionInformation/gmi:MI_AcquisitionInformation/gmi:instrument"
+    "/gmi:MI_Instrument"))
```

Comment:
Quotes should line up directly below the quote above.

Reply:
Done.